utils/check-lua-code-style.sh

```diff
@@ -1,15 +0,0 @@
-#!/bin/sh
```

Comment:
why do we need to delete this script?

one PR for one thing.

Reply:
Sorry, it's an accident.